### ISSUE TITLE
Run result of generator through jsonrepair

### ISSUE THREAD

**Issue by fcisio** [jondot/hygen#298]
Hey! I'm generating some `package.json` and the end result is pretty dirty.

I'm able to run `prettier` as a command with the `sh` option of the front-matter.
I would also like to run `jsonrepair`, which would allow not to write logic just for the commas in json.

`jsonrepair` doesn't have a CLI and I can't seem to find any alternative.

So I'm thinking it would be nice to be able to do something like:

```js
const jsonrepair = require('jsonrepair')

jsonrepair(generatorOutput)
```

This would also allow to cleanup pretty much everything with `prettier`.

---

As an alternative, I tried creating an helper, but there is no way to preserve `ejs` syntaxt, passing in in a function.

Possibly related to #296

**Comment by airtonix** [comment]
https://github.com/josdejong/jsonrepair#command-line-interface-cli

`4_finally_repair.ejs.t`
```
---
sh: jsonrepair broken.json > repaired.json
---
```

**Comment by fcisio** [comment]
@airtonix 
Yes! I created a feature request which was deved pretty quickly 👍

**Comment by stropho** [comment]
a workaround for me was to create my own node executable. Something like
```
#!/usr/bin/env node

const foo = require('my-favourite-lib')
// get something from `process.argv`

// do any operations you like
```
and then in the hygen template just
```
---
sh: path/to/my/executable.js
---
``` 
and if you are struggling with having the correct path to the executable, you can setup a hygen helper to always use the absolute path :crossed_fingers:
